### ISSUE TITLE
[Npc][Spell][Dungeon] Captain Skarloc Old Hillsbrad Foothills Normal

### ISSUE THREAD

**Issue by alanparrish** [Atlantiss/NetherwingBugtracker#7399]
[//]: # (Enclose links to things related to the bug using http://wowhead.com or any other TBC database.)
[//]: # (You can use screenshot ingame to visual the issue.)
[//]: # (Write your tickets according to the format:)
[//]: # ([Quest][Azuremyst Isle] Red Snapper - Very Tasty!)
[//]: # ([NPC] Magistrix Erona)
[//]: # ([Spell][Mage] Fireball)
[//]: # ([Npc][Drop] Ghostclaw Lynx)
[//]: # ([Web] Armory doesnt work)
[//]: # (To find server revision type `.server info` command in the game chat.)

**Description**: 

Captain Skarloc uses his "consecration" ability in normal mode.

**Current behaviour**:

Captain Skarloc uses his "consecration" ability in normal mode.

**Expected behaviour**: 

Captain Skarloc should only use his "consecration" ability in Heroic Mode. Links from TBC retail : 
https://wowwiki.fandom.com/wiki/Captain_Skarloc "Heroic Only

    Consecrate: Players standing on Consecrated ground take 675 Holy damage per second"

https://www.wowhead.com/npc=17862/captain-skarloc#comments

"On Heroic, the boss himself has only one serious threat, and that's his Consecration. Not only is it nasty, but he can cast two of them at once. If several members of your party are in it, your healer can't keep them all up.

Simply put, the tank must keep moving around to stay out of consecration. If the tank stays still, everyone gets hit by double consecration, and everyone dies. Melee attackers must not wait for the tank, but move out of it on their own." - 
by Kathucka (22,546 – 4·36·180) on 2008/05/22 (Patch 2.4.1)

**Server Revision**: 
3511

**Comment by Wolffenstein** [comment]
Fixed in rev 3523.
